sgx/ias-types/src/json.rs

```diff
@@ -0,0 +1,169 @@
+//! Basic RJSON DOM, taken from rjson tests.
```

Comment:
This file is basically copy-paste from attest, it provides a generic JSON DOM parser that elides the need for serde JSON.